tests/end-to-end/api/01-users.js

```diff
@@ -1741,4 +1760,49 @@ describe('[Users]', function() {
 				.end(done);
 		});
 	});
+
+	describe('[/users.logoutOtherClients]', () => {
+		let user;
+		let userCredentials;
+		let newCredentials;
+
+		before(async () => {
+			user = await createTestUser();
+			userCredentials = await loginTestUser(user);
+			newCredentials = await loginTestUser(user);
+		});
+		after(async () => {
+			await deleteTestUser(user);
+			user = undefined;
+		});
+
+		it('should invalidate all active sesions', (done) => {
+			/* We want to validate that the login with the "old" credentials fails
+			However, the removal of the tokens is done asynchronously.
+			Thus, we check that within the next seconds, at least one try to
+			access an authentication requiring route fails */
+			let counter = 0;
+
+			async function checkAuthenticationFails() {
+				const result = await request.get(api('me'))
+					.set(userCredentials);
+				return result.statusCode === 401;
+			}
+
+			async function tryAuthentication() {
+				if (await checkAuthenticationFails()) {
+					done();
+				} else if (++counter < 20) {
+					setTimeout(tryAuthentication, 1000);
```

Comment:
I wonder if this way the tests will not take too long to execute. :thinking:

Reply:
As stated in the code comments, the invalidation is done asychronously, so we need to wait. In the best case, it is only 1 second, but I have configured a timeout of 20 seconds as a maximum timeout to be sure that the test completes in a reasonable time.